changelog/_unreleased/2020-09-17-add-keyboard-shortcuts-for-reviews.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Add keyboard shortcuts for reviews
+issue: NEXT-9202
+author: max
```

Comment:
Can you please use the full name here? :-)

Reply:
Done